app/src/main/res/values/strings.xml

```diff
@@ -1,9 +1,9 @@
 <resources>
     <string name="app_name">Dice Roller</string>
     <string name="roll">Roll</string>
-    <string name="dice1">Come on, roll the dice!</string>
```

Comment:
The strings I removed in this file were not being used. Some of them were being used to set text in the layout but it was always overwritten when a state was emitted from the ViewModel. The values still exist as a developer tool but there is nothing to gain from keeping them here if they aren't user facing.